lib/widgets/action_sheet.dart

```diff
@@ -64,6 +64,16 @@ void _showActionSheet(
     });
 }
 
+/// A button in an action sheet.
+///
+/// When built from server data, the action sheet ignores changes in that data;
+/// we intentionally don't live-update the buttons on events.
+/// If a button's label, action, or position changes suddenly,
+/// it can be confusing and make the on-tap behavior unexpected.
+/// Better to let the user decide to tap
```

Comment:
Hmm, yeah.

Looking for `PerAccountStoreWidget.of` calls in this file, I think there's one place we don't currently follow this: the `hasSelfVote` in `ReactionButtons`.

That one shouldn't cause this problem in practice, though, because it changes by the user's own action. (Or by the message getting deleted, but then it doesn't matter whether the user tries to add a reaction or remove one.)